File: frontend/apps/admin/features/fa-im-pages/types/group/ImGroup.ts
// types/group/ImGroup.ts
export interface ImGroup {
  id: number;              // 主键ID
  groupId: string;         // 群ID
  groupName: string;       // 群名称
  groupAvatar?: string;    // 群头像
  groupNotice?: string;    // 群公告
  creatorId: number;       // 创建者ID
  maxMembers?: number;     // 最大成员数
  currentMembers?: number; // 当前成员数
  status?: number;         // 状态: 1正常 2解散
  deleted?: boolean;       // 删除状态
  crtTime?: string;        // 创建时间
  updTime?: string;        // 更新时间
}

File: fa-im/src/main/java/com/faber/api/im/group/entity/ImGroup.java
package com.faber.api.im.group.entity;

import com.faber.core.bean.BaseDelEntity;
import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * 群聊信息实体类
 *
 * @author faber
 */
@Data
@EqualsAndHashCode(callSuper = false)
@TableName("im_group")
public class ImGroup extends BaseDelEntity {
    /**
     * 群ID
     */
    @TableField("group_id")
    private String groupId;

    /**
     * 群名称
     */
    @TableField("group_name")
    private String groupName;

    /**
     * 群头像
     */
    @TableField("group_avatar")
    private String groupAvatar;

    /**
     * 群公告
     */
    @TableField("group_notice")
    private String groupNotice;

    /**
     * 创建者ID
     */
    @TableField("creator_id")
    private Long creatorId;

    /**
     * 最大成员数
     */
    @TableField("max_members")
    private Integer maxMembers;

    /**
     * 当前成员数
     */
    @TableField("current_members")
    private Integer currentMembers;
}